registry/txt_test.go

```diff
@@ -814,6 +819,15 @@ func testTXTRegistryMissingRecordsNoPrefix(t *testing.T) {
 				endpoint.OwnerLabelKey: "owner",
 			},
 		},
+		{
+			DNSName:       "oldformat3.test-zone.example.org",
+			SetIdentifier: "one",
+			Targets:       endpoint.Targets{"bar1.loadbalancer.com"},
+			RecordType:    endpoint.RecordTypeA,
+			Labels: map[string]string{
+				endpoint.OwnerLabelKey: "owner",
+			},
+		},
```

Comment:
I'm not sure which case is more nominal:
- the one you used here where you have 2 records with the same name but different identifiers coming from 2 owners
- the I thought of where you have 2 records with the same name bu different identifiers coming from the same owner

Maybe it makes sense to have both, WDYT?

Reply:
I don't think it makes sense.